README.md

```diff
@@ -70,7 +86,7 @@ The aws cli worker will accept a message matching to the following example:
 {
   "parameters": [
     {
-      "value": "aws2 s3 cp s3://mybucket/{element_filename} {target_filename}",
+      "value": "aws s3 cp s3://mybucket/{element_filename} {target_filename}",
```

Comment:
What the reason to not create a script and use credentials from messages (from VAULT).
Because now you have to configure AWS credentials many times ...

Reply:
TBD as it’s not possible right now, issues are openned.